hooks/pre-commit

```diff
@@ -8,22 +8,33 @@ PASS=true
 if [[ ("$STAGED_RS_FILES" != "") ]]; then
     printf "\nValidating Rust:\n"
 
-    cd driver
-    cargo clippy -- -D warnings
+    cargo clippy --all --all-targets -- -D warnings
 
     if [[ "$?" == 0 ]]; then
-      printf "\t\033[32mcargo clippy Passed: $FILE\033[0m"
+      printf "\t\033[32mcargo clippy Passed:\033[0m"
     else
-      printf "\t\033[41mcargo clippy Failed: $FILE\033[0m"
+      printf "\t\033[41mcargo clippy Failed: \033[0m Try running cargo clippy"
```

Comment:
ubernit: can you quote the command so that its clear what you are supposed to run. Also I would add the `--all --all-targets` so that its more clear what the exact command that failed is.